buildroot-external/patches/occu/0187-WebUI-Fix-GroupEditingIfDevicesInInbox/occu/WebUI/www/webui/js/common/viewmodels.js

```
@@ -0,0 +1,192 @@
+  self.getConfigPending = function()
+  {
+  var interfaceId = "BidCos-RF";
+  if(!self.type.startsWith("HM-"))
+  {
+    interfaceId = "HmIP-RF";
+  }
+  
+  var deviceParamSet = homematic('Interface.getParamset', {"interface": interfaceId, "address" : self.realSerialNumber+":0", "paramsetKey" : "VALUES"});
+    // conInfo(deviceParamSet);
+    // handle null/undefined like no config pending
+  if(deviceParamSet != typeof 'undefined' && deviceParamSet != null && (deviceParamSet.CONFIG_PENDING === '1' || deviceParamSet.CONFIG_PENDING === true))
```

Biome

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)